doc/kmonad.kbd.5

```diff
@@ -0,0 +1,299 @@
+.\" Automatically generated by Pandoc 2.11.4
```

Comment:
It is under the file format section since this is not about how to use kmonad as a cmd-line tool rather about how to **format a config file**, so I thought it would be more suitable there. If you think it should not be there it is totally okay.

I agree it is better to remove this file for now and tackle it later as a separate PR.

Reply:
Ah I see.  I was tripped up by it still being called `kmonad`.  When we later tackle this again we should remember to call it `kmonad-config` or something like that and have the `kmonad` page be the general commands one

Reply:
Resolved in 722615de83518045136f4fe0b9a7ce608cacac6c